examples/first-project/contracts/__upgradeable__/CounterUpgradeable.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.5.0;
```

Comment:
I think there's two questions and the answer is not necessarily the same. 1) Should we recommend that users commit these files to version control? 2) Should we commit these files in our integration tests?

I think the answer should be "no" to both, but @ylv-io thinks at least the first one should be "yes".

Reply:
Good point. I don't mind much about (2), but (1) is really important. 
I'm curious to hear _why_ you two think it should/shouldn't be committed.

Reply:
Just the general guideline that build artifacts shouldn't be committed, they should be rebuilt. But it's not a strong opinion in this case because it's new territory.

Reply:
I belive so because we want to keep an eye on any changes to Transpiler output so nothing can slip unnoticed.